### ISSUE TITLE
children route with replace donot work as expected in android4.3 webview

### ISSUE THREAD

**Issue by bajian** [vuejs/vue-router#1082]
router version:2.1.1
environment: in webview in android4.3 ,keep-alive route-view ,children route

now, I have a sub route in the `index page` ( contains children page a,children page b) and a `other page`
```
const index_children=[
{
    path:'a',
    component: (resolve) => {
        require(['./views/a.vue'], resolve)
    }
},
{
    path:'b',
    component: (resolve) => {
        require(['./views/b.vue'], resolve)
    }
},
{
    path:'',
    component: (resolve) => {
        require(['./views/a.vue'], resolve)
    }
}]


export default [
{
    path:'/index',
    meta: { requiresAuth: true },
    component: (resolve) => {
        require(['./views/index.vue'], resolve)
    },
    children:index_children
},
{
    path:'/other',
    component: (resolve) => {
        require(['./views/other.vue'], resolve)
    }
}
]
```
if I call `$router.replace('/index/b')` from route `'/index/a'` ,then call `$router.push('/other')` from  route `'/index/b'`,finally press back (history.back())
it goes back to  route `'/index/a'` as unexpected. in android4.3 webview
but it goes back to  route `'/index/b'` as expected. in android4.4 webview  and PC chrome

here is the code sample:
https://github.com/bajian/router-bug-sample

**Comment by posva** [comment]
If it got fixed in 4.4 it may be a webview bug and I think we may not do anything about it. I searched for that specific bug and for their bug tracker but couldn't find much.
@fnlctrl WDYT?

**Comment by fnlctrl** [comment]
Yeah.. I doubt if there's anything we can do to fix that. 

@bajian It would be great if you can provide some error logs for us to look into (if there were any), if not then I'm sorry there's nothing we can do since we don't have an android 4.3 device for testing. Closing now, but feel free to follow up with more information.
